src/measurement/gctx.rs

```diff
@@ -12,6 +13,14 @@ use crate::{
     measurement::snp::{SnpLaunchDigest, LD_BYTES},
 };
 
+#[cfg(not(feature = "openssl"))]
```

Comment:
I don't believe this is correct, as it likely wouldn't work when both `openssl` and `crypto_nossl` features are disabled.

```suggestion
#[cfg(all(not(feature = "openssl"), feature = "crypto_nossl")))]
```

Reply:
Is it a convention to always mention exhaustive conditions? The parent module already covers this case.